### PR TITLE
Changed ore spawn rates and added gravel to L5 and L2

### PR DESCRIPTION
**Pull request by VaIienta**

### CODE REVIEW

File: servers/survival/plugins/HiddenOre/config.yml [hunk 1]
```diff
@@ -55,67 +55,43 @@ regions:
...
-            package:
-            - ==: org.bukkit.inventory.ItemStack
-              type: REDSTONE_ORE
-            tools:
-            - anything
-            chance: 0.001
-            minAmount: 1
-            maxAmount: 4
-            transformIfAble: true
-            transformDropIfFails: false
```
Comment (Boy0000) [inline servers/survival/plugins/HiddenOre/config.yml:112]: Add redstone to layer 1 and 2. It is for depth meter recipe.

File: servers/survival/plugins/HiddenOre/config.yml [hunk 2]
```diff
@@ -55,67 +55,43 @@ regions:
...
-            package:
-            - ==: org.bukkit.inventory.ItemStack
-              type: GRAVEL
-            tools:
-            - anything
-            chance: 0.001
-            minAmount: 3
-            maxAmount: 12
-            transformIfAble: false
-            transformDropIfFails: false
```
Comment (Boy0000) [inline servers/survival/plugins/HiddenOre/config.yml:68]: Until mobspawning works better with hammerbeaks gravel should probably atleast have a chance to spawn. If not the getting flint will only happen in layer 2 for alot of people

File: servers/survival/plugins/HiddenOre/config.yml [hunk 3]
```diff
@@ -151,101 +127,88 @@ regions:
...
+            tansformIfAble: false
+            transformDropIfFails: false
           gold_ore:
             package:
-            - ==: org.bukkit.inventory.ItemStack
-              type: GOLD_ORE
+              - ==: org.bukkit.inventory.ItemStack
+                type: GOLD_ORE
             tools:
-            - anything
```
Comment (Boy0000) [inline servers/survival/plugins/HiddenOre/config.yml:204]: All these indents arent in the current one, but seem to be in the example config so shouldnt break anything

**Review by Boy0000 — CHANGES_REQUESTED** [review]

File: servers/survival/plugins/HiddenOre/config.yml (hunk 3 above)
Comment (VaIienta) [reply servers/survival/plugins/HiddenOre/config.yml:204]: not sure probably something when i copied it over

**Review by Boy0000 — APPROVED** [review]

**Pull request merged**